### PR TITLE
JC-2399 Fixing 500 error which occurs when redirecting to http error info pages

### PR DESCRIPTION
**Pull request by usharik**

### CODE REVIEW

File: jcommune-view/jcommune-web-controller/src/main/java/org/jtalks/jcommune/web/controller/ErrorsHandlerController.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+    @RequestMapping(value = "400")
+    @ResponseStatus(value = HttpStatus.BAD_REQUEST)
+    public String handleBadRequestError() {
+        return "/errors/400";
+    }
+
+    @RequestMapping(value = "403")
+    @ResponseStatus(value = HttpStatus.FORBIDDEN)
+    public String handleForbiddenError() {return "/errors/403";
```
Comment (ctapobep) [inline jcommune-view/jcommune-web-controller/src/main/java/org/jtalks/jcommune/web/controller/ErrorsHandlerController.java]: Formatting?

**Comment by ctapobep** [comment]
In the commit message there is no info about what the problem was and how the change fixes it..

File: jcommune-view/jcommune-web-controller/src/main/java/org/jtalks/jcommune/web/controller/ErrorsHandlerController.java [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+/**
+ * This controller is needed to handle errors and set correct status
+ *
+ * @author Andrey Ivanov
```
Comment (ctapobep) [inline jcommune-view/jcommune-web-controller/src/main/java/org/jtalks/jcommune/web/controller/ErrorsHandlerController.java]: Hm.. Was this copied from somewhere?

File: jcommune-view/jcommune-web-controller/src/main/java/org/jtalks/jcommune/web/interceptors/PropertiesInterceptor.java [hunk 3]
```diff
@@ -120,6 +120,11 @@ public void postHandle(HttpServletRequest request, HttpServletResponse response,
         }
     }
 
+    private boolean checkViewNameForErrorAndRedirection(ModelAndView modelAndView) {
```
Comment (ctapobep) [inline jcommune-view/jcommune-web-controller/src/main/java/org/jtalks/jcommune/web/interceptors/PropertiesInterceptor.java]: `isErrorOrRedirect()`? Shouldn't tests be written for the new logic?

File: jcommune-view/jcommune-web-controller/src/main/java/org/jtalks/jcommune/web/controller/ErrorsHandlerController.java (hunk 2 above)
Comment (usharik) [inline jcommune-view/jcommune-web-controller/src/main/java/org/jtalks/jcommune/web/controller/ErrorsHandlerController.java]: Yes. I use version of this controller which I remove in this PR
https://github.com/jtalks-org/jcommune/pull/269

**Comment by ctapobep** [comment]
@usharik there are still issues with this approach. If I open `/branches/100` I see 404 error with all the styles while if I open `/blah` I don't see any styles loaded.

**Comment by ctapobep** [comment]
Merged!

**Pull request closed without merging**